src/main/resources/ath-container/run.sh

```diff
@@ -74,6 +74,10 @@ fi
 
 shift 2
 
+if [[ $# -eq 0 ]]; then
+	extra_args+=' test'
+fi
+
 set -x
 
-BROWSER=$browser JENKINS_WAR=$war $MVN test $extra_args "$@"
+BROWSER=$browser JENKINS_WAR=$war $MVN $extra_args "$@"
```

Comment:
I needed to invoke this script with `process-test-resources` instead of `test` to download the WAR file, so making the `test` argument optional. To preserve compatibility I am setting it if it was not set by the caller, so the behavior is the same as before when no argument is passed in.